DocumentFormat.OpenXml/src/Framework/OpenXmlPackage.cs

```diff
@@ -1384,6 +1384,56 @@ internal class PartUriHelper
             private Dictionary<string, int> _sequenceNumbers = new Dictionary<string, int>(20);
             private Dictionary<string, int> _reservedUri = new Dictionary<string, int>();
 
+            //List of contentTypes that need to have a 1 appended to the name
+            //for the first item in the package
+            private static readonly HashSet<string> _numberedContentTypes = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```

Comment:
Got it. As someone not very familiar with the spec, I would find it helpful for the comment above the hashset to indicate that the numbers in the comments in the list refer to the section from the spec